armi/tests/test_plugins.py

```diff
@@ -21,6 +23,26 @@
 from armi import interfaces
 from armi import plugins
 from armi import settings
+from armi.bookkeeping.tests._constants import TUTORIAL_FILES
+from armi.bookkeeping.tests.test_historyTracker import runTutorialNotebook
+from armi.cases import case
+from armi.context import ROOT
+from armi.reactor.flags import Flags
+from armi.tests import ArmiTestHelper
+from armi.utils.directoryChangers import TemporaryDirectoryChanger
+
+CASE_TITLE = "anl-afci-177"
+TUTORIAL_DIR = os.path.join(ROOT, "tests", "tutorials")
+
+
+class TestPluginAfterLoadDB(plugins.ArmiPlugin):
```

Comment:
Maybe just remove `Test` from the class name because it's easy to miss that this doesn't inherit from the unit testing class in Python

Reply:
Sure, renamed.